VAMobile/e2e/tests/SettingsScreen.e2e.ts

```diff
@@ -61,6 +61,34 @@ describe('Settings Screen', () => {
     }
   })
 
+  it('should show Give feedback screen', async () => {
+    try {
+      await expect(element(by.text('Give feedback'))).toExist()
+    } catch (ex) {
+      await element(by.text('Developer Screen')).tap()
+      await element(by.text('Remote Config')).tap()
+      await waitFor(element(by.text('Override Toggles'))).toBeVisible().whileElement(by.id('remoteConfigTestID')).scroll(400, 'down')
+      await waitFor(element(by.text('inAppRecruitment'))).toBeVisible().whileElement(by.id('remoteConfigTestID')).scroll(100, 'down')
+      await element(by.text('inAppRecruitment')).tap()
+      await element(by.id('remoteConfigTestID')).scrollTo('bottom')
+      await element(by.text('Apply Overrides')).tap()
+      await loginToDemoMode()
+      await openProfile()
+      await openSettings()
+    }
+    await element(by.text('Give feedback')).tap()
+    await expect(element(by.text('Make this app better for all Veterans'))).toExist()
+    await expect(element(by.text('Go to questionnaire'))).toExist()
+    await expect(element(by.text('Learn more about the Veteran Usability Project'))).toExist()
```

Comment:
testID="inAppRecruitmentLearnMoreTestID" instead? Or could remove that testID from the InAppRecruitmentScreen file.